dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/event/csv/CsvEventService.java

```diff
@@ -30,18 +30,17 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.OutputStream;
-
-import org.hisp.dhis.dxf2.events.event.Events;
+import java.util.List;
 
 /**
  * @author Morten Olav Hansen <mortenoh@gmail.com>
  */
-public interface CsvEventService
+public interface CsvEventService<T>
```

Comment:
I might jave overlooked it, but is this Service used with another type than the org.hisp.dhis.tracker.domain.Event ?

Reply:
The other one is `org.hisp.dhis.dxf2.events.event.Event`